src/main/java/ui/TextUi.java

```diff
@@ -124,4 +124,23 @@ public void displayList(TaskList tasks){
         }
     }
 
+    public void findList(TaskList tasks, String item){
+        showToUser(
+                Message.MESSAGE_LINE,
+                Message.MESSAGE_FIND
+        );
+        int marker = 1;
+        String thisItem;
+        for(int i = 0; i < tasks.getList().size(); i++){
```

Comment:
Can consider using Java streams for easier processing. Additionally, it looks like your find command is capitalisation insensitive; is this intended?